apps/aecore/src/aec_trees.erl

```diff
@@ -623,8 +623,9 @@ apply_txs_on_state_trees([SignedTx | Rest], ValidTxs, InvalidTxs, Trees, Env, Op
                     Reason = {Type, What},
                     lager:error("Tx ~p cannot be applied due to an error ~p", [Tx, Reason]),
                     {error, Reason};
-                Type:What when not Strict ->
+                Type:What:_ST when not Strict ->
```

Comment:
I thought I'd verify that a stacktrace in fact won't be built with this pattern, but the compiler _is_ smart enough to figure that out.